.idea/compiler.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do not commit anything under .idea or target. and do not merge your PR before review

Reply:
create a new PR link:
https://github.com/MariaSarnova/FirstTask/pull/2